azure-iot-device/tests/iothub/pipeline/test_pipeline_stages_iothub_http.py

```diff
@@ -3,34 +3,23 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-import functools
 import logging
 import pytest
 import json
 import sys
 import six.moves.urllib as urllib
-from azure.iot.device.common.pipeline import (
-    pipeline_events_base,
-    pipeline_ops_base,
-    pipeline_stages_base,
-    pipeline_ops_http,
-)
```

Comment:
Cool!  How did you know that these imports were unused?

Reply:
I have an unused import warning level set in Pycharm i think..I set it long back...

---
In reply to: [384156889](https://github.com/Azure/azure-iot-sdk-python/pull/464#discussion_r384156889) [](ancestors = 384156889)